bin/deploy-rabbit

```diff
@@ -4,7 +4,14 @@
 SCRIPT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 MODE=${MODE:-uaa}
-CONFIG=${CONFIG:-rabbitmq.config}
+
+if [[ $OSTYPE == 'darwin'* ]]; then
+  DEFAULT_CONFIG=rabbitmq_macos.config
```

Comment:
Which docker product are you using in your mac? I am using Docker Desktop 4.6.0. I do not have any issues with 0.0.0.0. I am bit reluctant to accept this change. I have not heard of anyone having this issue.

Reply:
MacOs Ventura, Docker Desktop >4.15.
I installed also telnet in the rabbitmq container to verify if I was wrong.
If you prefer, I can also remove the configuration from macOS just leaving the note in the readme in case someone falls in the same trap.
I think this is due to some limitation/security enforcement on macOS, I haven't dig that much.
```
root@98b0aef7b4b9:/# telnet 0.0.0.0 8080
Trying 0.0.0.0...
telnet: Unable to connect to remote host: Connection refused
root@98b0aef7b4b9:/# 
```

I'm going to disable Virtualization framework just to check if this could be the problem.

Reply:
Also with virtualisation framework disabled it is not working. 

However, I think the problem is mostly related by the fact that both rabbitmq-server and management ui reaches the same domain to reach Keycloak.

As an example, rabbitmq management ui could be reachable from clients but then rabbitmq-server could not be allowed to access the same domain accessible by the users (ex. its in a militarized zone where the access to the internet is not allowed)
Of course there are some workarounds to that, like defining same domains with different IPs depending which dns is asked for the information, but it is quite complicated to replicate this scenario in some usecases.
What do you think?
Anyway, let me know if you prefer me to remove this specific thing for macOS, I'll add it as a side note in the readme in the case.

Reply:
Do not revert those changes just yet. Once I finish looking at the other issue I get back to this one. Thanks.